packages/mds-metrics-sheet/metrics-log.ts

```diff
@@ -143,11 +144,12 @@ async function getProviderMetrics(iter: number): Promise<MetricsSheetRow[]> {
           inactive: 0,
           elsewhere: 0
         }
-        if (last[provider.provider_id].event_counts_last_24h) {
-          event_counts = last[provider.provider_id].event_counts_last_24h
-          status_counts = eventCountsToStatusCounts(last[provider.provider_id].event_counts_last_24h)
-          starts = last[provider.provider_id].event_counts_last_24h.trip_start || 0
-          ends = last[provider.provider_id].event_counts_last_24h.trip_end || 0
+        const { event_counts_last_24h } = last[provider.provider_id]
+        if (event_counts_last_24h) {
+          event_counts = event_counts_last_24h
+          status_counts = eventCountsToStatusCounts(event_counts_last_24h)
+          starts = event_counts_last_24h.trip_start || 0
+          ends = event_counts_last_24h.trip_end || 0
```

Comment:
This is where you'd want to extract the `trip_enter`s I think, avoids the need for lodash. Then simply stick that var in (potentially an undefined var) in the return statement!